tox.ini

```diff
@@ -0,0 +1,45 @@
+# For more information about tox, see https://tox.readthedocs.io/en/latest/
+[tox]
+envlist = py{38,39,310}-{linux,macos,windows}-{pyqt,pyside}
+toxworkdir=/tmp/.tox
+isolated_build = true
+
+[gh-actions]
+python =
+    3.8: py38
+    3.9: py39
+    3.10: py310
+    
+[gh-actions:env]
+PLATFORM =
+    ubuntu-latest: linux
+    macos-latest: macos
+    windows-latest: windows
+
+[testenv]
+platform = 
+    macos: darwin
+    linux: linux
+    windows: win32
+setenv =
+    PYTHONPATH = {toxinidir}
+passenv = 
+    CI
+    GITHUB_ACTIONS
+    DISPLAY
+    XAUTHORITY
+    NUMPY_EXPERIMENTAL_ARRAY_FUNCTION
+    PYVISTA_OFF_SCREEN
+conda_deps =
+    # use conda to install numcodecs on mac py3.9
+    py39-macos: numcodecs
+conda_channels =
+    conda-forge
+deps = 
+    pytest  # https://docs.pytest.org/en/latest/contents.html
+    pytest-cov  # https://pytest-cov.readthedocs.io/en/latest/
+    pytest-qt  # https://pytest-qt.readthedocs.io/en/latest/intro.html
+    pytest-xvfb ; sys_platform == 'linux'
+    pyqt: napari[pyqt5,testing]
+    pyside: napari[pyside2,testing]
```

Comment:
Why use old napari for these tests? It should test from the current `main` at this moment.

Reply:
The setup mimics what is being done at [`napari-console`](https://github.com/napari/napari-console/blob/main/tox.ini), since I am not too familiar with Tox. Can I just put GitHub urls here for `main` or how does that work?

Reply:
Yes. Such line should work:
`git+https://github.com/napari/napari`

Here example of usage: https://github.com/4DNucleome/PartSeg/blob/17eb8128e65f099536ac3edf2394b17b9b03787c/tox.ini#L78

When we release napari which supports plugin manager as an external package, we need to test against a main branch and stable release.